ecl/hqlcpp/hqlttcpp.cpp

```diff
@@ -5078,6 +5080,10 @@ void GlobalAttributeInfo::extractStoredInfo(IHqlExpression * expr, IHqlExpressio
             getStringValue(s, codehash);
             storedName.setown(createConstant(s.str()));
         }
+        if (expr->hasAttribute(refreshAtom))
```

Comment:
I don't think this is needed given that getBoolValue is passed the default? I think queryAttributeChild returns NULL if the searched for atom is not present.